django_prices_vatlayer/european_vat.py

```diff
@@ -0,0 +1,34 @@
+from decimal import Decimal
+from prices import Tax, PriceRange
+
+from .utils import get_tax_for_country
+
+
+class EuropeanVAT(Tax):
+    def __init__(self, country_code, rate_name=None):
+        self.country_code = country_code
+        self.rate_name = rate_name
+        self.vat_rate = None
```

Comment:
It should be possible to set a tax rate even without the API, so I'd recommend to add `vat_rate` parameter to the constructor. It should be not required.
As a bonus, every result of `__repr__` pasted into interpreter will produce valid PriceModifier instance.